python/pfs/utils/constants.py

```diff
@@ -0,0 +1,18 @@
+"""General PFS engineering constants
+"""
+N_SPECTROGRAPHS = 4  # Number of spectrographs
+FIBERS_PER_SPECTROGRAPH = 651  # Number of fibers per spectrograph
+N_SUNSS_FIBERS_INST = 127  # Number of SuNSS fibers per instrument
+N_FIELDS_PFI = 3  # Number of fields in the PFI
+N_MODULES_PER_FIELD = 14  # Number of modules in a PFI field
+N_COBRAS_PER_MODULE = 57  # Maxmimum number of cobras in a given module.
+N_SCIENCE_FIBERS = 2394  # Number of science fibers.
```

Comment:
Where do we need this?

Reply:
Removed - had assumed that other PFS software might use this, but I agree they shouldn't from a design standpoint.